pkgs/applications/graphics/gimp/plugins/default.nix

```diff
@@ -23,7 +23,10 @@ let
   }
   // a
     # don't call this gimp-* unless you want nix replace gimp by a plugin :-)
-  // { name = "${a.name}-${gimp.name}-plugin"; }
+  // {
+      name = "${a.name}-${gimp.name}-plugin";
+      buildInputs = [ gimp gimp.gtk glib pkgconfig ] ++ (a.buildInputs or []);
```

Comment:
Thinked about split  nativeBuildInputs, but was urge to rebuild my system. Will do.